### PR TITLE
Update webpack config to use babel-loader on vue & tag files

### PR DESCRIPTION
**Pull request by brainbrian**
- include node_module based code in the transpilation, takes longer but
  ensures our code is built to our standards

Issue: #37

### CODE REVIEW

File: webpack-config/webpack.base.conf.js [hunk 1]
```diff
@@ -78,8 +78,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js|\.jsx$/,
-        exclude: /(node_modules)/,
+        test: /\.js|\.jsx|\.vue|\.tag$/,
```
Comment (peripateticus) [inline webpack-config/webpack.base.conf.js]: While you're in here, could you delete the `.jsx` test.

File: webpack-config/webpack.base.conf.js [hunk 2]
```diff
@@ -78,29 +78,11 @@ module.exports = {
...
-                      'Firefox > 57',
-                      'iOS > 7',
-                      'Safari >= 9',
-                      'Explorer >= 11',
-                      'Edge >= 15',
-                    ],
-                  },
-                },
-              ],
-              babelPresetES2015Riot,
```
Comment (peripateticus) [inline webpack-config/webpack.base.conf.js]: Looks like we no longer need this `require`: https://github.com/rei/febs/blob/975c9bb1370ddbf157db23258ffe6fe921eb6057/webpack-config/webpack.base.conf.js#L12

**Comment by peripateticus** [comment]
This is great, thank you @brainbrian!

File: webpack-config/webpack.base.conf.js (hunk 2 above)
Comment (brainbrian) [reply webpack-config/webpack.base.conf.js]: Cleaned up 2 requires that were moved to the .babelrc file

Comment (peripateticus) [inline webpack-config/webpack.base.conf.js]: Also, we need to bump the patch version.

Comment (brainbrian) [reply webpack-config/webpack.base.conf.js]: I assume that's `v2.4.3` then? Upgraded from `v2.4.2`

Comment (peripateticus) [reply webpack-config/webpack.base.conf.js]: Yep!

Comment (brainbrian) [reply webpack-config/webpack.base.conf.js]: done.

**Pull request merged**